scripts/consolidate-memory.py

```
@@ -0,0 +1,492 @@
+            entry['task_id'] = task_id_match.group(1)
+
+        # Parse outcome
+        outcome_match = re.search(r'\*\*Outcome\*\*:\s*(\w+)', section)
+        if outcome_match:
+            entry['outcome'] = outcome_match.group(1)
+
+        # Parse learnings
+        learnings_match = re.search(r'\*\*Learnings\*\*:\n((?:- .+\n?)+)', section)
+        if learnings_match:
+            learnings = re.findall(r'- (.+)', learnings_match.group(1))
+            entry['learnings'] = [l for l in learnings if l not in ('None documented', 'None')]
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)